build-tools/packages/build-tools/src/fluidBuild/fluidBuild.ts

```diff
@@ -141,9 +141,7 @@ async function main() {
 
 	const timeInMinutes =
 		timer.getTotalTime() > 60000
-			? ` (${Math.floor(timer.getTotalTime() / 60000)}m ${(
-					(timer.getTotalTime() % 60000) / 1000
-				).toFixed(3)}s)`
+			? ` (${Math.floor(timer.getTotalTime() / 60000)}m ${((timer.getTotalTime() % 60000) / 1000).toFixed(3)}s)`
 			: "";
 	log(`Total time: ${(timer.getTotalTime() / 1000).toFixed(3)}s${timeInMinutes}`);
```

Comment:
My eyeballs can't pinpoint anything more than a formatting change here...
Can we not call a time related function up to 4 times and assume it returns the same value every time? Gotta grab such things locally unless you want it to be variable.

Reply:
I removed this change and will follow up with a separate PR.

Reply:
https://github.com/microsoft/FluidFramework/pull/22834